<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Download</class>
 <widget class="QDialog" name="Dialog_Download">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Firmware</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rrcc.qrc">
    <normaloff>:/png/png/save.png</normaloff>:/png/png/save.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Download Parameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox_model">
          <property name="toolTip">
           <string>Select Model</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Generation 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Generation 2</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="toolTip">
           <string>Enter Firmware</string>
          </property>
          <property name="text">
           <string notr="true">v11_00????.fullos.pkg</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_server">
        <property name="toolTip">
         <string>Select Server</string>
        </property>
        <item>
         <property name="text">
          <string notr="true">cdn.awsbj0.fds.api.mi-img.com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">cdn.awsde0.fds.api.mi-img.com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">cdn.cnbj0.fds.api.mi-img.com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">cdn.cnbj2.fds.api.mi-img.com</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="toolTip">
         <string>Download Progress</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_model</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>comboBox_server</tabstop>
 </tabstops>
 <resources>
  <include location="../rrcc.qrc"/>
 </resources>
 <connections/>
</ui>
